shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,75 @@
+
+## Node Hierarchy
+As Decred builds out its infrastructure, it becomes apparent that the underlying security is a high priority. Each time a new service is added, it increases the opportunity for node participation. This is done on multiple levels, and it’s important to note that not every node is equal.
+
+**A top tier fully validating node** is a dcrd unit run either as a stand-alone piece of software or as part of a service for 24 hours, 7 days a week, 365 days a year. This node has an inbound port open, so it can be publicly accessible for other nodes to connect to. This full node is the one that adds security to the chain and makes the project more decentralised. 
+
+**Side note: **There is also an option for running dcrd over the Tor network for greater privacy and security.
+
+**A second tier fully validating node** is a dcrd unit running intermittently or continuously, with the default settings which don’t include an open inbound port. This is not a publicly accessible node, which means other nodes in the network can’t connect to it, but it can connect to them. This full node is perfect for validating chain information and is good for personal privacy and security. This node setup doesn’t add to the security of the chain or make the project more decentralised.
+
+**Simple Payment Verification (SPV)** is not a node, but a wallet function. It allows the use of a Decred wallet without having to download the entire Decred blockchain. A wallet operating in SPV mode only needs to download full blocks containing transactions relevant to it and block headers. This reduces the wallet’s hardware requirements and greatly reduces the initial load time for new wallets. When an SPV wallet initialises, it will connect to the Decred network using dcrd peer connections.
+
+## Example:
+When you run dcrd for the first time, it needs to figure out where to look for the Blockchain, so it can start downloading blocks. dcrd has a hardcoded list of seeders and when you start dcrd for the first time it will query the seeders for IPs of known good nodes and then connect to those and starts downloading blocks.
+
+But then with each connection it makes, the nodes talk and share their connections, like saying, "hey here are all the IP addresses I know of who run dcrd." And so, your node will start building its own list of dcrd peers to check and ask, "hey, do any of you have any new blocks?" And it can do that without accepting new inbound connections. That's how it works for people who don't want to open up their inbound connection.
+
+But if nobody opened up their inbound connection through their firewall, then all these dcrd nodes would be reaching out to the IP addresses in their peers list and asking "hey, can I talk to you? Do you have new blocks?" And the firewall would just block all the traffic and no peers would be able to talk to each other.
```

Comment:
`However, imagine if nobody opened up inbound connections through their firewall. Every dcrd node would just continually reach out to the IP addresses in their peers list, asking "Hey, can I talk to you? Do you have new blocks?" And each firewall would just block all traffic. No nodes would be able to talk to each other.`